apis/stash/v1beta1/backup_blueprint_types.go

```diff
@@ -48,7 +48,11 @@ type BackupBlueprint struct {
 type BackupBlueprintSpec struct {
 	// RepositorySpec is used to create Repository crd for respective workload
 	v1alpha1.RepositorySpec `json:",inline"`
-	Schedule                string `json:"schedule,omitempty"`
+	// RepoNamespace specifies the namespace where the Repository will be created for the respective target
+	// +optional
+	RepoNamespace string `json:"repoNamespace,omitempty"`
+	// Schedule specifies the schedule for invoking backup sessions
```

Comment:
// Schedule specifies the default schedule for backup.
// You can overwrite this schedule for a particular target using `<schedule annotation>` annotation.